### PR TITLE
Improved ESH-INF labels to comply with casing rules

### PR DESCRIPTION
**Pull request by Hilbrand**
See: https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

### CODE REVIEW

**Comment by wborn** [comment]
Thanks for the huge effort @Hilbrand. Do you also have plans to update the English (Dutch?) i18n properties files?

File: bundles/org.openhab.binding.evohome/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="account" />
 		</supported-bridge-type-refs>
-		<label>evohome Display</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.evohome/src/main/resources/ESH-INF/thing/thing-types.xml]: In this case its a product name, however, they are not too good anyway in promoting the correct usage

File: bundles/org.openhab.binding.sonyaudio/src/main/resources/ESH-INF/thing/srs-zr5.xml [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 	<!-- SRS-ZR5 Thing Type -->
 	<thing-type id="SRS-ZR5">
-		<label>SONY wirelesspeaker SRS-ZR5</label>
+		<label>SONY Wirelesspeaker SRS-ZR5</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.sonyaudio/src/main/resources/ESH-INF/thing/srs-zr5.xml]: This should actually be two words...

File: bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/bridge.xml [hunk 3]
```diff
@@ -5,32 +5,32 @@
...
 				max="49151">
 				<context>network-address</context>
-				<label>Optolink adapter port</label>
+				<label>Optolink Adapter Port</label>
 				<description>Port of the LAN gateway</description>
 				<default>31113</default>
 			</parameter>
 			<parameter name="adapterID" type="text" required="true">
-				<label>adapterID</label>
+				<label>AdapterID</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/bridge.xml]: Two words

File: bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/bridge.xml [hunk 4]
```diff
@@ -5,32 +5,32 @@
...
 				<description>The ID of the adapter.</description>
 			</parameter>
 			<parameter name="refreshInterval" type="integer" required="false" min="60"
 				max="600">
 				<context>refresh</context>
-				<label>refreshInterval</label>
+				<label>RefreshInterval</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/bridge.xml]: Two words

File: bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 5]
```diff
@@ -131,7 +131,7 @@
 				</properties>
 			</channel>
 			<channel id="timer_TU" typeId="timer">
-				<label>Tuesday heatingtimer</label>
+				<label>Tuesday Heatingtimer</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/thing-types.xml]: Heating Timer

Comment (martinvw) [reply bundles/org.openhab.binding.vitotronic/src/main/resources/ESH-INF/thing/thing-types.xml]: x10 >

**Comment by Hilbrand** [comment]
@wborn I haven't looked into the properties files yet, but intended to do so. These files also might be looked at for correct encoding.

File: bundles/org.openhab.io.homekit/src/main/resources/ESH-INF/config/config.xml [hunk 6]
```diff
@@ -7,18 +7,18 @@
...
 			<description>General thermostat settings</description>
 		</parameter-group>
 		<parameter-group name="thermostatTargetHeatingCooling">
-			<label>Thermostat targetHeatingCooling mapping</label>
+			<label>Thermostat TargetHeatingCooling Mapping</label>
 			<description>String values used by your thermostat to set different targetHeatingCooling modes</description>
 		</parameter-group>
 		<parameter-group name="thermostatCurrentHeatingCooling">
-			<label>Thermostat currentHeatingCooling mapping</label>
+			<label>Thermostat CurrentHeatingCooling Mapping</label>
```
Comment (martinvw) [inline bundles/org.openhab.io.homekit/src/main/resources/ESH-INF/config/config.xml]: ....

File: bundles/org.openhab.binding.buienradar/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 7]
```diff
@@ -75,168 +75,168 @@
...
 		<item-type>Number:Speed</item-type>
-		<label>Current rainfall</label>
+		<label>Current Rainfall</label>
 		<description>Current rainfall</description>
 		<state readOnly="true" pattern="%.10f %unit%" />
 	</channel-type>
 
 	<channel-type id="forecast_5">
 		<item-type>Number:Speed</item-type>
-		<label>Rainfall in 5 min</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.buienradar/src/main/resources/ESH-INF/thing/thing-types.xml]: This is a unit and it is officially lower case - so I would keep it as is.

File: bundles/org.openhab.binding.digiplex/src/main/resources/ESH-INF/thing/area.xml [hunk 8]
```diff
@@ -23,7 +23,7 @@
 		</channels>
 		<config-description>
 			<parameter name="refreshPeriod" type="integer">
-				<label>Refresh time of area status (in seconds)</label>
+				<label>Refresh Time of Area Status (in Seconds)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.digiplex/src/main/resources/ESH-INF/thing/area.xml]: The `(in Second)` should be completely removed from the label. It can be mentioned in the description and even better by adding the `unit=` attribute to the parameter.

File: bundles/org.openhab.binding.evohome/src/main/resources/ESH-INF/thing/thing-types.xml (hunk 1 above)
Comment (kaikreuzer) [reply bundles/org.openhab.binding.evohome/src/main/resources/ESH-INF/thing/thing-types.xml]: Yes, I remember this from the review - it should be kept lower case as the product name.

File: bundles/org.openhab.binding.folding/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 9]
```diff
@@ -30,7 +30,7 @@
 				<context>password</context>
 			</parameter>
 			<parameter name="polling" type="integer">
-				<label>Polling interval (s)</label>
+				<label>Polling Interval (s)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.folding/src/main/resources/ESH-INF/thing/thing-types.xml]: remove `(s)`

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-poller.xml [hunk 10]
```diff
@@ -8,11 +8,11 @@
 			<bridge-type-ref id="serial" />
 		</supported-bridge-type-refs>
 
-		<label>Regular poll</label>
+		<label>Regular Poll</label>
 		<description>Regular poll of data from Modbus slaves</description>
 		<config-description>
 			<parameter name="refresh" type="integer" min="0">
-				<label>Poll interval (ms)</label>
+				<label>Poll Interval (ms)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-poller.xml]: remove the `(ms)`

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-poller.xml [hunk 11]
```diff
@@ -38,12 +38,12 @@
...
 			</parameter>
 			<parameter name="maxTries" type="integer" min="1">
-				<label>Maximum tries when reading</label>
+				<label>Maximum Tries When Reading</label>
 				<default>3</default>
 				<description>Number of tries when reading data, if some of the reading fail. For single try, enter 1.</description>
 			</parameter>
 			<parameter name="cacheMillis" type="integer" min="0">
-				<label>Cache duration (ms)</label>
+				<label>Cache Duration (ms)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-poller.xml]: remove `(ms)`

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml [hunk 12]
```diff
@@ -94,13 +94,13 @@
 				<advanced>true</advanced>
 			</parameter>
 			<parameter name="receiveTimeoutMillis" type="integer" min="0">
-				<label>Read operation timeout (ms)</label>
+				<label>Read Operation Timeout (ms)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml]: remove `(ms)`

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml [hunk 13]
```diff
@@ -124,18 +124,18 @@
 
 			<!-- connection handling -->
 			<parameter name="timeBetweenTransactionsMillis" type="integer" min="0">
-				<label>Time between transactions (ms)</label>
+				<label>Time Between Transactions (ms)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml]: remove `(ms)`

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml [hunk 14]
```diff
@@ -124,18 +124,18 @@
...
 			<parameter name="connectMaxTries" type="integer" min="1">
-				<label>Maximum connection tries</label>
+				<label>Maximum Connection Tries</label>
 				<description>How many times we try to establish the connection. Should be at least 1.</description>
 				<default>1</default>
 				<advanced>true</advanced>
 			</parameter>
 			<parameter name="connectTimeoutMillis" type="integer" min="0">
-				<label>Timeout for establishing the connection (ms)</label>
+				<label>Timeout for Establishing the Connection (ms)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml]: remove `(ms)`

Comment (kaikreuzer) [reply bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-serial.xml]: and also for all further ones...

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-tcp.xml [hunk 15]
```diff
@@ -25,40 +25,40 @@
...
 			<parameter name="connectMaxTries" type="integer" min="1">
-				<label>Maximum connection tries</label>
+				<label>Maximum Connection Tries</label>
 				<description>How many times we try to establish the connection. Should be at least 1.</description>
 				<default>1</default>
 				<advanced>true</advanced>
 			</parameter>
 			<parameter name="reconnectAfterMillis" type="integer" min="0">
-				<label>Disconnect and establish the connection again after this time (ms)</label>
+				<label>Disconnect and Establish the Connection Again After This Time (ms)</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-tcp.xml]: That's a good example why we should think about a strict length limit for labels :-)

File: bundles/org.openhab.binding.network/src/main/resources/ESH-INF/binding/binding.xml [hunk 16]
```diff
@@ -10,25 +10,25 @@
...
 			<default>true</default>
-			<label>Listen for dhcp requests</label>
+			<label>Listen for DHCP Requests</label>
 			<description>Usually a device requests an IP address in an IPv4 network with the help of DHCP as soon as it enters a network. If we listen to those
 			packets, we can detect a device presence even faster. You need elevated access rights (see readme) for this to work.</description>
 		</parameter>
 		<parameter name="cacheDeviceStateTimeInMS" type="integer">
 			<default>2000</default>
-			<label>Cache time in ms</label>
+			<label>Cache Time in ms</label>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.network/src/main/resources/ESH-INF/binding/binding.xml]: remove `in ms`

File: bundles/org.openhab.binding.network/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 17]
```diff
@@ -41,17 +41,17 @@
 				<default>1</default>
 			</parameter>
 
-			<parameter name="timeout" type="integer">
-				<label>Timeout in ms</label>
-				<description>States how long to wait for a response (in MS), before if a device is stated as offline</description>
+			<parameter name="timeout" type="integer" unit="ms">
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.network/src/main/resources/ESH-INF/thing/thing-types.xml]: 👍

File: bundles/org.openhab.binding.kodi/src/main/resources/ESH-INF/thing/thing-types.xml [hunk 18]
```diff
@@ -133,19 +133,19 @@
...
 	<channel-type id="playfavorite">
 		<item-type>String</item-type>
-		<label>Play or open a Favorite</label>
+		<label>Play or Open a Favorite</label>
 		<description>Play or open the given favorite by sending a command with the favorite's title</description>
 	</channel-type>
 	<channel-type id="pvr-open-tv" advanced="true">
 		<item-type>String</item-type>
-		<label>Play PVR tv</label>
+		<label>Play PVR Tv</label>
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.kodi/src/main/resources/ESH-INF/thing/thing-types.xml]:
```suggestion
		<label>Play PVR TV</label>
```

File: bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/thing/thermostat.xml [hunk 19]
```diff
@@ -10,7 +10,7 @@
 			<bridge-type-ref id="netatmoapi" />
 		</supported-bridge-type-refs>
 
-		<label>Thermostat relay/plug</label>
+		<label>Thermostat Relay/plug</label>
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.netatmo/src/main/resources/ESH-INF/thing/thermostat.xml]:
```suggestion
		<label>Thermostat Relay/Plug</label>
```

**Comment by cweitkamp** [comment]
One more question: Did you apply all the changes by hand or by using a tool? I am asking because we might want to check OHC too. We have a lot of configuration, default channel types, profiles and automation actions, conditions and triggers over there.

> I haven't looked into the properties files yet, but intended to do so. These files also might be looked at for correct encoding.

The new `mvn spotless:apply` command will take care of file encodings. But I did not test it for I18n files yes.

File: bundles/org.openhab.binding.yeelight/src/main/resources/ESH-INF/thing/ct_bulb.xml [hunk 20]
```diff
@@ -5,7 +5,7 @@
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
 	<thing-type id="ct_bulb">
-		<label>Yeelight White LED Bulb v2</label>
+		<label>Yeelight White LED Bulb V2</label>
```
[the remark this answers is not in the document]
Comment (claell) [reply bundles/org.openhab.binding.yeelight/src/main/resources/ESH-INF/thing/ct_bulb.xml]: I'd agree, although I am not 100% sure about the official wording.

**Comment by Hilbrand** [comment]
>  Did you apply all the changes by hand or by using a tool?
@cweitkamp I used a combination of a simple shell script and manual editing for the edge cases. And I have a pr for SAT to test for this, which I used to check if I didn't miss anything.

File: bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-tcp.xml (hunk 15 above)
Comment (Hilbrand) [reply bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-tcp.xml]: I already added such a check to the new label SAT check I'm implementing. The question is what would be a good length limit?

**Comment by martinvw** [comment]
@Hilbrand do you believe you have processed all coments? So good to merge if the build is fine?

File: bundles/org.openhab.binding.allplay/src/main/resources/ESH-INF/binding/binding.xml [hunk 21]
```diff
@@ -8,13 +8,13 @@
 
 	<config-description>
 		<parameter name="rewindSkipTimeInSec" type="integer">
-			<label>Rewind skip time (s)</label>
+			<label>Rewind Skip Time (s)</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.allplay/src/main/resources/ESH-INF/binding/binding.xml]: o o

File: bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/config/config.xml [hunk 22]
```diff
@@ -44,11 +44,11 @@
 
 	<config-description uri="enocean:virtualRockerChannel:config">
 		<parameter name="duration" type="integer">
-			<label>Time in ms between Press and Release message</label>
+			<label>Time Between Press and Release Message (in ms)</label>
```
Comment (martinvw) [inline bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/config/config.xml]: ms

**Comment by Hilbrand** [comment]
I won't claim to have fixed all unit cases. But my main focus for this pr was the casing of the label words. There are several other xml related topics to address and I plan to do more. In that context I'll hope to also look into the unit part.

For the labels I think I've covered all (except Tesla, but that is due to #6342)

**Comment by martinvw** [comment]
@kaikreuzer you have requested changes, okay to merge?

**Comment by martinvw** [comment]
@Hilbrand I'm okay with merging, is the broken test for homie convention also there in master?

**Comment by Hilbrand** [comment]
@martinvw yes it also seems to be broken in master.
